syn/ibex_core_lr_synth_conf.tcl

```diff
@@ -0,0 +1,51 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# List of inputs and outputs.  Number is timing constraint expressed as a % of
+# clock cycle, e.g.
+#
+# {instr_req_o 60.0}
+#
+# as an output means the intstr_req_o output must be stable by 60% of total
+# clock cycle
+#
+# {instr_gnt_i 30.0}
+#
+# as an input means the instr_gnt_i input will be stable by 30% of the total
+# clock cycle
+
+set lr_synth_outputs [list {instr_req_o   70.0} \
+                           {instr_addr_o  70.0} \
+                           {data_req_o    70.0} \
+                           {data_we_o     70.0} \
+                           {data_be_o     70.0} \
+                           {data_addr_o   70.0} \
+                           {data_wdata_o  70.0} \
+                           {core_sleep_o  80.0}]
+
+set lr_synth_inputs [list {test_en_i      0.0 } \
+                          {hart_id_i      0.0 } \
+                          {boot_addr_i    0.0 } \
+                          {instr_gnt_i    30.0} \
+                          {instr_rvalid_i 30.0} \
+                          {instr_rdata_i  30.0} \
+                          {instr_err_i    30.0} \
+                          {data_gnt_i     30.0} \
+                          {data_rvalid_i  30.0} \
+                          {data_rdata_i   30.0} \
+                          {data_err_i     30.0} \
+                          {irq_software_i 10.0} \
+                          {irq_timer_i    10.0} \
+                          {irq_external_i 10.0} \
+                          {irq_fast_i     10.0} \
+                          {irq_nm_i       10.0} \
+                          {debug_req_i    10.0} \
+                          {fetch_enable_i 0.0 }]
+
+# clock and reset IO names (at top-level)
+set lr_synth_clk_input clk_i
+set lr_synth_rst_input rst_ni
+
+# clock period in ps
+set lr_synth_clk_period 3000.0
```

Comment:
Is there a motivation for this clock period?

Reply:
It's a bit faster than we can achieve with nangate45 and it's generally useful to push beyond what you can achieve to see where the limit is.

I've changed it to 4000.0 (250 Mhz) as that gets closer to meeting timing and does meet reg2reg timing as ultimately Ibex probably isn't a core where you're trying to clock it as fast as possible so better to give it an easier target.